py/py_sparkling/ml/algos.py

```diff
@@ -28,7 +30,42 @@ def propagate_value_from_deprecated_property(kwargs, from_deprecated, to_replaci
         del kwargs[from_deprecated]
 
 
-class H2OGBM(H2OGBMParams, JavaEstimator, JavaH2OMLReadable, JavaMLWritable):
+class H2OAlgorithmCommons(JavaEstimator, JavaH2OMLReadable, JavaMLWritable):
+
+    def getBinaryModel(self):
+        model_id = py4j.java_gateway.get_field(self._java_obj.getBinaryModel(), "_key").toString()
+
+        model_json = h2o.api("GET /%d/Models/%s" % (3, model_id))["models"][0]
+        metrics_class, model_class = H2OEstimator._metrics_class(model_json)
+        m = model_class()
+        m._id = model_id
+        m._model_json = model_json
+        m._have_pojo = model_json.get('have_pojo', True)
```

Comment:
Do we need a pojo?

Reply:
This is based on the H2O's code -> wanted to keep parity of the information

Reply:
ok, no problem :-)